python/ray/_raylet.pyx

```diff
@@ -2732,6 +2736,58 @@ cdef void terminate_asyncio_thread() nogil:
         core_worker = ray._private.worker.global_worker.core_worker
         core_worker.stop_and_join_asyncio_threads_if_exist()
 
+
+cdef void _cleanup_actor_instance_wrapper() noexcept nogil:
```

Comment:
I would not prefix this with `_` because it is called from outside this call (via callback from C++)

Let's also "call it what it is": `call_actor_shutdown` and then the internal one can be `_call_actor_shutdown`